packages/terra-form-select/src/single/Menu.jsx

```diff
@@ -229,16 +229,25 @@ class Menu extends React.Component {
       }
     }
 
+    const optGroupElement = MenuUtil.getOptGroupElement(this.props.children, this.state.active);
     const element = MenuUtil.findByValue(this.props.children, this.state.active);
+
+    let displayText;
+    if (optGroupElement) {
+      displayText = `Group ${optGroupElement.props.label}, ${element.props.display}`;
```

Comment:
Does this need to be fully translated?

Reply:
Added here 7defe48c45eaa2e6265e7dfea86ffe9496c3afea
Type fixed here 9c75500046b25e53bb41fc7c6419936021756488